lib/service/state.go

```diff
@@ -35,6 +37,15 @@ const (
 	stateDegraded = 2
 )
 
+var stateGauge = prometheus.NewGauge(prometheus.GaugeOpts{
+	Name: teleport.MetricState,
+	Help: "State of the teleport process: 0 - degraded, 1 - recovering, 2 - ok",
```

Comment:
According to https://github.com/gravitational/teleport/blob/0780b517715d1b3fa3122acef946f8c9c0be85bf/lib/service/state.go#L28-L38 - these are the other way round:

0 - ok
1 - recovering
2 - degraded

Reply:
Good catch, fixed